biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/WriteDependencies.scala

```diff
@@ -16,10 +16,14 @@ package nl.lumc.sasc.biopet.core
 
 import java.io.{File, PrintWriter}
 
+import akka.actor.ActorSystem
```

Comment:
Yes, this seems like overkill indeed. But also want to see if this framework can be used later to make live reporting while running. Then such a framework is needed anyway